cmake/ConfigCPack.cmake

```
@@ -173,6 +173,7 @@ if(ROCPROFSYS_USE_MPI)
 endif()
 if(ROCPROFSYS_BUILD_TESTING)
     list(APPEND _DEBIAN_PACKAGE_DEPENDS "rocdecode-test")
+    list(APPEND _DEBIAN_PACKAGE_DEPENDS "rocjpeg")
```

```suggestion
    list(APPEND _DEBIAN_PACKAGE_DEPENDS "rocjpeg-test")
```
Should it be rocjpeg-test?